### ISSUE TITLE
@astrojs/lit | this._$Ej.forEach is not a function

### ISSUE THREAD

**Issue by andeeplus** [withastro/astro#8825]
### Astro Info

```block
Astro                    v3.3.0
Node                     v18.17.1
System                   macOS (x64)
Package Manager          npm
Output                   static
Adapter                  none
Integrations             @astrojs/lit
```

### Describe the Bug

Starting a new project from scratch I'm not able to render lit elements in any way except using the `client:only="lit"` directive.

- https://docs.astro.build/en/install/auto/
- https://docs.astro.build/en/guides/integrations-guide/lit/#installation

**Error:** `this._$Ej.forEach is not a function`

`TypeError: this._$Ej.forEach is not a function
    at MyElement.update (file:///Users/username/Desktop/test/node_modules/@lit/reactive-element/node/reactive-element.js:1:5579)
    at LitElementRenderer.connectedCallback (file:///Users/username/Desktop/test/node_modules/@lit-labs/ssr/lib/lit-element-renderer.js:51:42)
    at render (/Users/username/Desktop/test/node_modules/@astrojs/lit/server.js:62:11)
    at render.next (<anonymous>)
    at Object.renderToStaticMarkup (/Users/username/Desktop/test/node_modules/@astrojs/lit/server.js:108:11)
    at renderFrameworkComponent (/Users/username/Desktop/test/node_modules/astro/dist/runtime/server/render/component.js:205:66)
    at async Module.renderComponent (/Users/username/Desktop/test/node_modules/astro/dist/runtime/server/render/component.js:356:10)`

### What's the expected result?

Render the component without issue.

### Link to Minimal Reproducible Example

https://stackblitz.com/edit/github-fqvkdx?file=src%2Fpages%2Findex.astro

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by lilnasy** [comment]
Your reproduction installs Lit 3.0. It is not supported yet.
- https://github.com/withastro/roadmap/discussions/728

**Comment by andeeplus** [comment]
@lilnasy I didn't install Lit, I've just used `npx astro add lit`.
Probably this is the issue.

**Comment by lilnasy** [comment]
Ok, it seems to install 3.0 by default. As a wrokaround, you can ensure the version in your package json is exactly "2.8.0".

**Comment by lilnasy** [comment]
`@lit-labs/ssr-client` has started requiring lit 3.0. Annoyingly, this breaking change was made in a minor release. Our adapter  will have to pin it to 1.0 to unbreak builds.

**Comment by andeeplus** [comment]
@lilnasy Glad you find a quick fix, let's wait for the lit 3.0 support then, thanks
